app/components/review_summary/view.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module ReviewSummary
+  class View < GovukComponent::Base
+    renders_one :header
+
+    def initialize(form:, trainee:, data_view: ApplyInvalidDataView.new(trainee.apply_application))
+      @has_errors = form.respond_to?(:errors) ? form.errors.any? : false
```

Comment:
You can get rid of this line and use what's suggested below for `errors_captured?`